lib/rules/web/admin_console/4.0/common_ui_elements.xyaml

```diff
@@ -62,6 +62,14 @@ uncheck_checkbox:
     selector:
       xpath: //input[contains(@type,'checkbox')]
     op: click
+check_link_and_text:
+  element:
+    selector:
+      xpath: //a[@class='co-resource-link__resource-name' and text()='<text>' and @href='<link_url>']
```

Comment:
Do we need use parameter for @class? eg @class='classname'

Reply:
I think we don't, even I'm thinking if we need remove `@class='co-resource-link__resource-name'` from xpath expression, it turns out to check only an `//a` element containing desired `text` and `link`, WDYT?

Reply:
Agree, class is not needed.

Reply:
Updated